myapp/migrations/1666549806352133_matches.postgres.up.sql

```diff
@@ -7,5 +7,7 @@ CREATE TABLE matches (
     percent_match float,
     song_id integer,
     created_at timestamp without time zone NOT NULL DEFAULT now(),
-    expiry timestamp without time zone NOT NULL DEFAULT now() + interval '5 day'
+    expiry timestamp without time zone NOT NULL DEFAULT now() + interval '5 day',
+    user_a_viewed BOOLEAN NOT NULL DEFAULT FALSE,
+    user_b_viewed BOOLEAN NOT NULL DEFAULT FALSE
```

Comment:
The changes needs to be in a new migration- These won't be run on systems that are currently active, the ID for this migration has already been used